src/helpers/utils.ts

```diff
@@ -46,14 +46,19 @@ export function sanitizeSubscriptions(list?: string | string[]) {
 
 export async function subscribe(email: string, address: string) {
   const subscriber = { email, address, created: currentTimestamp() };
-  return await db.queryAsync('INSERT IGNORE INTO subscribers SET ?', [subscriber]);
+  await db.queryAsync('INSERT IGNORE INTO subscribers SET ?', [subscriber]);
+
+  return await db.queryAsync(
```

Comment:
why do you need to query a newly created subscriber? If you want to understand whether a new row was inserted or not, you can try `affectedRows` in response to the query